src/storage/size-reporter/FileSizeReporter.ts

```diff
@@ -0,0 +1,53 @@
+import { existsSync, readdirSync, statSync } from 'fs';
+import { join } from 'path';
+import type { ResourceIdentifier } from '../../ldp/representation/ResourceIdentifier';
+import type { FileIdentifierMapper } from '../mapping/FileIdentifierMapper';
+import type { Size } from './size.model';
+import type { SizeReporter } from './SizeReporter';
+
+/**
+ * SizeReporter that is used to calculate sizes of resources for a file based system
+ */
+export class FileSizeReporter implements SizeReporter {
+  // The FileSizeReporter will always return byte values
+  public unit = 'bytes';
```

Comment:
Why public? Should be read-only in any case. But might not need the variable.

Reply:
It might be usefull for other classes to see what kind of data the SizeReporter will return ( Thinking about some admin interface ).

Reply:
See https://github.com/solid/community-server/pull/973#discussion_r723328105 .
And indeed forgot to make it readonly!